src/test/java/spark/utils/MimeParseTest.java

```diff
@@ -4,11 +4,200 @@
 
 import java.util.Arrays;
 import java.util.Collection;
-
+import com.diffblue.deeptestutils.Reflector;
+import org.junit.Assert;
 import static org.junit.Assert.*;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.HashMap;
+
 
 public class MimeParseTest {
 
+  @Rule public ExpectedException thrown = ExpectedException.none();
```

Comment:
Are we happy keeping this level of reflection for tests that access private methods?

Reply:
I will remove this test class just to be super clean